src/utils/validationSchema.ts

```diff
@@ -0,0 +1,75 @@
+import * as yup from 'yup';
```

Comment:
I now know what yup is I thought zod was the better choice here, can you please double check this? I would rather use zod since it's probably better supported and more common. What are your thoughts?